src/wazuh_db/helpers/wdb_agents_helpers.h

```diff
@@ -47,4 +48,18 @@ int wdb_agents_vuln_cve_insert(int id,
 int wdb_agents_vuln_cve_clear(int id,
                               int *sock);
 
+/** 
+ * @brief Updates all or a specific status from the vuln_cve table in the agents database. 
+ *  
+ * @param[in] id The agent ID.
+ * @param[in] old_status The status that is going to be updated.
```

Comment:
It would be interesting if the helper description mentions the `old_status ="*"` option

Reply:
Added!